examples/light-switch-app/efr32/src/AppTask.cpp

```diff
@@ -221,9 +226,23 @@ void AppTask::SwitchActionEventHandler(AppEvent * aEvent)
     if (aEvent->Type == AppEvent::kEventType_Button)
     {
         BindingCommandData * data = Platform::New<BindingCommandData>();
-        data->commandId           = chip::app::Clusters::OnOff::Commands::Toggle::Id;
         data->clusterId           = chip::app::Clusters::OnOff::Id;
 
+        if (mCurrentButtonState)
```

Comment:
Shouldn't we define the mCurrentButtonState based on the binded light state?

Reply:
This is a problem that needs to be solved on the controller side. A switch could be binded to multiples lights, so in this case which light would be the right one to base our value ? This variable is more to represent the state of a "normal" switch on the wall, either it's in On position or in Off position (up or down).

Reply:
Yeah ok, but this way there is a 50% chance that the first time you press the button it doesn't "nothing". So how does it work currently with multiple bound devices? To whom is the command sent